tests/components/azure_event_hub/test_init.py

```diff
@@ -90,128 +38,178 @@ async def test_full_config(hass):
             },
         }
     }
-    config[azure_event_hub.DOMAIN].update(MIN_CONFIG)
-    assert await async_setup_component(hass, azure_event_hub.DOMAIN, config)
-
+    config[DOMAIN].update(CS_CONFIG_FULL)
+    assert await async_setup_component(hass, DOMAIN, config)
 
-async def _setup(hass, mock_call_later, filter_config):
-    """Shared set up for filtering tests."""
-    config = {azure_event_hub.DOMAIN: {"filter": filter_config}}
-    config[azure_event_hub.DOMAIN].update(MIN_CONFIG)
 
-    assert await async_setup_component(hass, azure_event_hub.DOMAIN, config)
-    await hass.async_block_till_done()
-    mock_call_later.assert_called_once()
-    return mock_call_later.call_args[0][2]
+async def test_filter_only_config(hass):
+    """Test the popping of the filter and further import of the config."""
+    config = {
+        DOMAIN: {
+            "filter": {
+                "include_domains": ["light"],
+                "include_entity_globs": ["sensor.included_*"],
+                "include_entities": ["binary_sensor.included"],
+                "exclude_domains": ["light"],
+                "exclude_entity_globs": ["sensor.excluded_*"],
+                "exclude_entities": ["binary_sensor.excluded"],
+            },
+        }
+    }
+    assert await async_setup_component(hass, DOMAIN, config)
 
 
-async def _run_filter_tests(hass, tests, process_queue, mock_batch):
-    """Run a series of filter tests on azure event hub."""
-    for test in tests:
-        hass.states.async_set(test.id, STATE_ON)
-        await hass.async_block_till_done()
-        await process_queue(None)
+async def test_unload_entry(hass, entry, mock_create_batch):
+    """Test being able to unload an entry.
 
-        if test.should_pass:
-            mock_batch.add.assert_called_once()
-            mock_batch.add.reset_mock()
-        else:
-            mock_batch.add.assert_not_called()
+    Queue should be empty, so adding events to the batch should not be called, this verifies that the unload, calls async_stop, which calls async_send and shuts down the hub.
```

Comment:
Please break long docstrings around 72 characters per line. The first line should be a header then a blank line and the rest of the docstring as body.